tests/test_workspacebuilder.py

```diff
@@ -101,12 +101,15 @@ def test_focus_pane_index(session):
     assert w.name != "man"
 
     pane_path = "/usr"
+    p = None
 
-    while retry():
+    def f():
+        nonlocal p
```

Comment:
Do you use `nonlocal` often? I am not familiar with it.

Reply:
I don't really use it often. But in this case it was an easy way to proceed since the previous code would modify an outer-scope variable (not a global, not a local).

Reply:
👍

Reply:
Sounds good to me!  Let's get it in, and I will publish another alpha release.

Reply:
(and ideally, a final release)